test/Fixtures/GetWithQuery.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Fixtures.GetWithQuery
+  ( GetWithQuery (..)
+  , TestQuery (..)
+  , getWithQueryOpenApiRouter
+  ) where
+
+import Beeline.Params ((?+))
+import Beeline.Params qualified as BP
+import Beeline.Routing ((/-), (/:))
+import Beeline.Routing qualified as R
+import Data.Int qualified as Int
+import Data.Text qualified as T
+import Shrubbery qualified as S
+
+import Fixtures.NoPermissions as Export
+import Orb qualified
+import TestDispatchM qualified as TDM
+
+getWithQueryOpenApiRouter :: Orb.OpenApiProvider r => r (S.Union '[GetWithQuery])
+getWithQueryOpenApiRouter =
+  Orb.provideOpenApi "get-with-query"
+    . R.routeList
+    $ Orb.get (R.make GetWithQuery /- "test" /- "get_with_query")
+      /: R.emptyRoutes
+
+data GetWithQuery = GetWithQuery
+
+data TestQuery = TestQuery
+  { textQueryParam :: T.Text
+  , int8QueryParam :: Maybe Int.Int8
+  }
+
+testQuerySchema :: BP.QuerySchema schema => schema TestQuery TestQuery
+testQuerySchema =
+  BP.makeParams TestQuery
+    ?+ BP.required textQueryParam (BP.textParam "textQueryParam")
+    ?+ BP.optional int8QueryParam (BP.int8Param "int8QueryParam")
+
+instance Orb.HasHandler GetWithQuery where
+  type HandlerRequestBody GetWithQuery = Orb.NoRequestBody
+  type HandlerRequestQuery GetWithQuery = TestQuery
+  type HandlerRequestHeaders GetWithQuery = Orb.NoRequestHeaders
```

Comment:
Are `HandlerRequestBody` and `HandlerRequestHeaders` really needed here? It looks like in a [test above](https://github.com/flipstone/orb/pull/29/files#diff-f475c6b2daebacf5e8f4efc70159783683dde1ecc93bbed9fd96a8f47c5a577aR40) we don't include `HandlerRequestBody` or `HandlerRequestQuery` so I am just curious.